Testscripts/Linux/LIS-MODULES-CHECK.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the Apache License.
 
 from azuremodules import *
+from distutils.version import LooseVersion
```

Comment:
I'm concerned with availibility of this module across the distros. Please execute and confirm.

Reply:
Verified  LooseVersion is available across distros